### PR TITLE
DB-16761

### PR DESCRIPTION
**Pull request by jasonar81**

### CODE REVIEW

File: src/main/java/com/ocient/cli/CLI.java [hunk 1]
```diff
@@ -2138,43 +2161,54 @@ private static boolean sourceCommands(final BufferedReader reader) throws IOExce
...
 					if (!comment && quote == '\0' && cmd.trim().endsWith(";"))
 					{
+						// System.out.println("Finished scrubbing command: '" + cmd + "'");
 						cmd = cmd.trim();
 						cmd = cmd.substring(0, cmd.length() - 1).trim();
 						if (trace && !endsWithIgnoreCase(cmd, " trace"))
 						{
 							cmd = cmd + " trace";
 						}
+						// System.out.println("Finished trimming scrubbed: '" + cmd + "'");
```
Comment (tli-ocient) [inline src/main/java/com/ocient/cli/CLI.java:2177]: Did you mean to clean this up?

**Review by tli-ocient — CHANGES_REQUESTED** [review]
I don't think you need to do another version bump since we didnt put 2.18 into db yet.

File: src/main/java/com/ocient/cli/CLI.java [hunk 2]
```diff
@@ -2149,32 +2173,41 @@ private static boolean sourceCommands(final BufferedReader reader) throws IOExce
...
 							cmd = cmd + " trace";
 						}
+
 						break;
 					}
 					else
 					{
-						final String cont = reader.readLine();
-						if (cont == null)
+						// System.out.println("Current command text: '" + cmd + "'");
```
Comment (tli-ocient) [inline src/main/java/com/ocient/cli/CLI.java:2181]: this too

File: src/main/java/com/ocient/cli/CLI.java (hunk 1 above)
Comment (jasonar81) [reply src/main/java/com/ocient/cli/CLI.java]: Done

File: src/main/java/com/ocient/cli/CLI.java (hunk 2 above)
Comment (jasonar81) [reply src/main/java/com/ocient/cli/CLI.java:2181]: Done

**Pull request merged**